tf_files/aws/modules/eks/cloud.tf

```diff
@@ -718,3 +718,60 @@ resource "null_resource" "config_setup" {
     "aws_autoscaling_group.eks_autoscaling_group",
   ]
 }
+
+
+
+#--------------------------------------------------------------
+# let's work towards EKS IAM-ServiceAccount integration
+
+resource "aws_iam_openid_connect_provider" "identity_provider" {
```

Comment:
Very cool!  Do we need to add some kind of check to remind the user to update EKS to 1.13+ as part of this?

Reply:
I couldn't find and elegant way to do it, but I'll try something before merging.

Reply:
1.11 is coming EOL soon, and 1.12 might also go away in the near future, instead of adding controls that won't make any difference soon we shouldn't spend more time on this.